File: Java/program59.java
import java.util.*;
 
 class Number
 {
    public  boolean CheckEven(int iNo1)
    {
        if((iNo1 % 2 ) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
 }
class program59
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        boolean bRet = false;

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();
      


        Number eobj =new Number();

        bRet = eobj.CheckEven(iValue);

        if(bRet == true)
        {
        System.out.println(iValue +"  is Even ");
        }
        else
        {
            System.out.println(iValue+"  is Odd ");
        }

    }
}

File: C/program59.c
#include<stdio.h>
#include<stdbool.h>

bool CheckEven(int iNo)
{
    if((iNo % 2) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }

}

int main()
{
    int iValue = 0;
    bool bRet = false;

    printf("Enter number :");
    scanf("%d",&iValue);

    bRet = CheckEven(iValue);

    if(bRet == true)
    {
        printf("%d is even \n",iValue);
    }
    else
    {
        printf("%d is odd \n",iValue);  
    }
    return 0;
}